JQuery/Custom-JQuery.js

```diff
@@ -0,0 +1,293 @@
+            if ((typeof propName) == 'string') {
+                this.each((item) => {
+                    getStyle(item, propName, properties);
+                });
+                return properties;
+            } else if ((typeof propName) == 'object') {
+                this.each((item) => {
+                    for (let key in propName) {
+                        setStyle(item, key, propName[key]);
+                    }
+                });
+            }
+        } else if (arguments.length == 2) {
+            if ((typeof propValue) == 'function') {
+                this.each((item) => {
+                    setStyle(item, propName, propValue());
+                });
+            } else if ((typeof propValue) == 'string') {
+                this.each((item) => {
+                    setStyle(item, propName, propValue);
+                });
+            }
+        }
+    }
+
+    JQuery.prototype.css = cssFunction;
+
+    let emptyFunction = function() {
+        this.each((item) => {
+            item.innerHTML = "";
+        })
+    }
+
+    JQuery.prototype.empty = emptyFunction;
+
+    let wrapFunction = function() {
+        const wrappingElementTag = arguments[0];
+        if (typeof wrappingElementTag == 'string') {
+            this.each((item) => {
+                appendAndWrapElement(item, wrappingElementTag);
+            });
+        } else if (typeof wrappingElement == 'function') {
+            this.each((item) => {
+                appendAndWrapElement(item, wrappingElementTag());
+            });
+        }
+    }
+
+    JQuery.prototype.wrap = wrapFunction;
+
+    //don't check
+    let toggleFunction = function() {
+        if (arguments.length == 0) {
+            this.each((item) => {
+                item.setAttribute('display', window.getComputedStyle(item).display)
+                if (window.getComputedStyle(item).display == 'none')
+                    item.style.display = item.getAttribute('display');
+                else if (item.getAttribute('display') == 'none')
+                    item.style.display = "initial";
+                else if (item.getAttribute('display') != 'none')
+                    item.style.display = "none";
+            });
+        } else if (arguments.length == 1 && typeof arguments[0] == 'boolean') {
+            const flag = arguments[0];
+            this.each((item) => {
+                if (flag == true)
+                    item.style.visibility = "visible";
+                else item.style.visibility = "hidden";
+            });
+        }
+    }
+
+    JQuery.prototype.toggle = toggleFunction;
+    return instantiate;
+})();
+
+function appendAndWrapElement(item, wrappingElementTag) {
+    item.insertAdjacentHTML('beforebegin', wrappingElementTag);
+    let wrappingElement = item.previousElementSibling;
+    wrappingElement.appendChild(item);
+}
+
+function getStyle(item, propName, properties) {
+    const styles = getComputedStyle(item);
+    for (let x in styles) {
```

Comment:
You can just use `styles[propName]` with an additional check that this field exists.